templates/contract-deploy/run-contract-setup.sh

```diff
@@ -137,6 +137,46 @@ jq --slurpfile c combined.json '.L1Config.polygonRollupManagerAddress = $c[0].po
 jq --slurpfile c combined.json '.L1Config.polTokenAddress = $c[0].polTokenAddress' genesis.json > g.json; mv g.json genesis.json
 jq --slurpfile c combined.json '.L1Config.polygonZkEVMAddress = $c[0].rollupAddress' genesis.json > g.json; mv g.json genesis.json
 
+# Create cdk-erigon node configs
+jq_script='
+.genesis | map({
+  (.address): {
+    contractName: (if .contractName == "" then null else .contractName end),
+    balance: (if .balance == "" then null else .balance end),
+    nonce: (if .nonce == "" then null else .nonce end),
+    code: (if .bytecode == "" then null else .bytecode end),
+    storage: (if .storage == null or .storage == {} then null else (.storage | to_entries | sort_by(.key) | from_entries) end)
+  }
+}) | add'
+
+# Use jq to transform the input JSON into the desired format
+output_json=$(jq "$jq_script" /opt/zkevm/genesis.json)
+
+# Handle jq errors
+if [ $? -ne 0 ]; then
+    echo "Error processing JSON with jq"
+    exit 1
+fi
```

Comment:
Wow, very thorough. I appreciate the error checking even if it's `[...]` rather than `[[...]]``